### ISSUE TITLE
Only color is used to show the data in the chart: A11y_.NET 9 template_Home_Use of Color.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#26801]
### GitHub Tags
#A11yTCS; #A11ySev2; #MAC; #WCAG1.4.1; #GH_.NET9template_MAC_Dec2024; #DesktopApp; #.NET Core; ##Use of Color; #A11yMAS;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
MAC Version 15.2

### Repro Steps:
Install and open ".NET 9 template" app.
"Home Screen" will open.
Tab till chart section.
### Actual Result:
only color is used to show the data in the chart.

### Expected Result:
Only color shouldn't be used to indicate data in the chart.

### User impact:
Color blinded users won't be able to know what lines showing which information in the chart.

### Version with bug

9.0.21 SR2.1

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

macOS

### Relevant log output

```shell

```

**Comment by InduPriya1805** [comment]
Rev:Inse

**Comment by sheiksyedm** [comment]
Duplicae #26236

**Comment by AnKushSingh05** [comment]
> Duplicae [#26236](https://github.com/dotnet/maui/issues/26236)





Hi @sheiksyedm, the bug you provided is for Win32, while the logged bug is for the MAC platform, so they are not duplicates.

**Comment by sheiksyedm** [comment]
@jfversluis @PureWeen  There is currently no way to display data labels directly on the radial bar chart, leaving tooltips as the only method to view category values. Additionally, showing the count value in the legend is not ideal, as the legend's primary purpose is to identify category bars based on color. Do you have any thoughts or suggestions on addressing these limitations?

About data label: https://help.syncfusion.com/maui-toolkit/circular-charts/datalabels#enable-data-labels

**Comment by tj-devel709** [comment]
> [@jfversluis](https://github.com/jfversluis) [@PureWeen](https://github.com/PureWeen) There is currently no way to display data labels directly on the radial bar chart, leaving tooltips as the only method to view category values. Additionally, showing the count value in the legend is not ideal, as the legend's primary purpose is to identify category bars based on color. Do you have any thoughts or suggestions on addressing these limitations?
> 
> About data label: https://help.syncfusion.com/maui-toolkit/circular-charts/datalabels#enable-data-labels

Yes, I am planning on changing this to a doughnut chart that has data labels to fix these accessibility issues with the radial chart
